cmd/sfncli/test_scripts/touch_file.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+mkdir -p $WORK_DIR
```

Comment:
is it up to the command to make the work dir? Is this what gearcmd does?

Reply:
Changed this behavior to check that we have the right access and then create a temporary directory per run and clean it up after the run: https://github.com/Clever/sfncli/pull/9/commits/d5f77d08d488f03303accdcf13fbd39cd86d0dc2

This ensures that `cmd` can assume that the directory exists and will be cleaned up. Also we validate/create the `workDirectory` on startup to ensure that permission issues show up before a task is run.